src/BooleanParamConverter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace ParamConverter;
+
+use Cake\Core\App;
+use Cake\Http\Exception\BadRequestException;
+use Cake\ORM\Entity;
+use Cake\ORM\TableRegistry;
+use Cake\Utility\Inflector;
+
+/**
+ * Class EntityParamConverter
+ *
+ * Param Converter for Entity classes
+ *
+ * @package ParamConverter
+ */
+class BooleanParamConverter implements ParamConverterInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function supports(string $class): bool
+    {
+        return $class === 'bool';
```

Comment:
@Ali1 my concern here is that we don't differentiate between built-in scalar types and class names.

Reply:
classes with names int bool string float bool and null are forbidden in PHP 7 so I feel that's good enough. You reckon a second argument would be better?